awx/api/serializers.py

```diff
@@ -3915,7 +3915,8 @@ class WorkflowJobNodeSerializer(LaunchConfigurationBaseSerializer):
     class Meta:
         model = WorkflowJobNode
         fields = ('*', 'credential', 'job', 'workflow_job', '-name', '-description', 'id', 'url', 'related',
-                  'unified_job_template', 'success_nodes', 'failure_nodes', 'always_nodes',)
+                  'unified_job_template', 'success_nodes', 'failure_nodes', 'always_nodes',
+                  'do_not_run',)
```

Comment:
Is this worth exposing in the API, or is it an internal field we only care about in the context of the task scheduler graph?

Reply:
I think it's worth showing to clients, because it's fundamentally core to how our form of convergence works. I can picture the questions now "the first node was successful, but the next node refuses to start!", okay, well it's waiting for the other branch to converge. You know if the branch is finished if there is a finished job or the DNR flag is true. Otherwise, what do you tell them?

That said, this needs help text. The help text should be added in the field on the model.